app/connection.go

```diff
@@ -77,29 +77,24 @@ func defaultDialOptions(c *cli.Context, addr *url.URL) ([]grpc.DialOption, error
 	return opts, nil
 }
 
-func newRPCCredential(c *cli.Context) (credentials.PerRPCCredentials, error) {
-	insecure := c.Bool(InsecureConnectionFlagName)
+func newRPCCredential(ctx *cli.Context) (credentials.PerRPCCredentials, error) {
+	insecure := ctx.Bool(InsecureConnectionFlagName)
 
-	apiKey := c.String(APIKeyFlagName)
+	apiKey := ctx.String(APIKeyFlagName)
 	if len(apiKey) > 0 {
 		return apikey.NewCredential(
 			apiKey,
 			apikey.WithInsecureTransport(insecure),
 		)
 	}
 
-	tokens, err := loadLoginConfig(c)
+	config, err := LoadTokenConfig(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	if len(tokens.AccessToken) > 0 {
-		return oauth.NewCredential(
-			tokens.AccessToken,
-			oauth.WithInsecureTransport(insecure),
-		)
-	}
-
-	// Use no credentials for this connection.
```

Comment:
was there any scenario where we wanted to use "no credentials" here? (e.g. development or w/e)

Reply:
There is no scenario, I just put this there a while back to pass nothing if nothing was specified